app-admin/kubelet-wrapper/files/kubelet-wrapper

```diff
@@ -1,39 +1,75 @@
+	for rev in $@ ; do
+		if [[ ! -z "${!rev}" ]]; then
+			return
+		fi
+	done
+	echo One of $@ must be set
+	exit 1
+}
+
+if [[ -n "${KUBELET_VERSION}" ]]; then
+	echo KUBELET_VERSION environment variable is deprecated, please use KUBELET_IMAGE_TAG instead
 fi
 
-KUBELET_ACI="${KUBELET_ACI:-quay.io/coreos/hyperkube}"
+if [[ -n "${KUBELET_ACI}" ]]; then
+	echo KUBELET_ACI environment variable is deprecated, please use the KUBELET_IMAGE_URL instead
+fi
+
+KUBELET_IMAGE_TAG="${KUBELET_IMAGE_TAG:-${KUBELET_VERSION}}"
+
+require_ev_one KUBELET_IMAGE KUBELET_IMAGE_TAG
+
+KUBELET_IMAGE_URL="${KUBELET_IMAGE_URL:-${KUBELET_ACI:-quay.io/coreos/hyperkube}}"
+KUBELET_IMAGE="${KUBELET_IMAGE:-${KUBELET_IMAGE_URL}:${KUBELET_IMAGE_TAG}}"
+
+if [[ "${KUBELET_IMAGE%%/*}" == "quay.io" ]]; then
+	RKT_RUN_ARGS="${RKT_RUN_ARGS} --trust-keys-from-https"
+fi
 
 mkdir --parents /etc/kubernetes
 mkdir --parents /var/lib/docker
 mkdir --parents /var/lib/kubelet
 mkdir --parents /run/kubelet
 
-exec /usr/bin/rkt run \
-  --volume etc-kubernetes,kind=host,source=/etc/kubernetes \
-  --volume etc-ssl-certs,kind=host,source=/usr/share/ca-certificates \
-  --volume var-lib-docker,kind=host,source=/var/lib/docker \
-  --volume var-lib-kubelet,kind=host,source=/var/lib/kubelet \
-  --volume os-release,kind=host,source=/usr/lib/os-release \
-  --volume run,kind=host,source=/run \
-  --mount volume=etc-kubernetes,target=/etc/kubernetes \
-  --mount volume=etc-ssl-certs,target=/etc/ssl/certs \
-  --mount volume=var-lib-docker,target=/var/lib/docker \
-  --mount volume=var-lib-kubelet,target=/var/lib/kubelet \
-  --mount volume=os-release,target=/etc/os-release \
-  --mount volume=run,target=/run \
-  --trust-keys-from-https \
-  $RKT_OPTS \
-  --stage1-from-dir=stage1-fly.aci \
-  ${KUBELET_ACI}:${KUBELET_VERSION} --exec=/kubelet -- "$@"
-
+RKT="${RKT:-/usr/bin/rkt}"
+RKT_STAGE1_ARG="${RKT_STAGE1_ARG:---stage1-from-dir=stage1-fly.aci}"
+set -x
+exec ${RKT} ${RKT_GLOBAL_ARGS} \
+	run ${RKT_RUN_ARGS} \
+	--volume etc-kubernetes,kind=host,source=/etc/kubernetes,readOnly=false \
+	--volume etc-ssl-certs,kind=host,source=/usr/share/ca-certificates,readOnly=true \
+	--volume usr-share-certs,kind=host,source=/usr/share/ca-certificates,readOnly=true \
+	--volume var-lib-docker,kind=host,source=/var/lib/docker,readOnly=false \
+	--volume var-lib-kubelet,kind=host,source=/var/lib/kubelet,readOnly=false \
+	--volume os-release,kind=host,source=/usr/lib/os-release,readOnly=true \
+	--volume run,kind=host,source=/run,readOnly=false \
+	--mount volume=etc-kubernetes,target=/etc/kubernetes \
+	--mount volume=etc-ssl-certs,target=/etc/ssl/certs \
+	--mount volume=usr-share-certs,target=/usr/share/ca-certificates \
+	--mount volume=var-lib-docker,target=/var/lib/docker \
+	--mount volume=var-lib-kubelet,target=/var/lib/kubelet \
+	--mount volume=os-release,target=/etc/os-release \
+	--mount volume=run,target=/run \
+	${RKT_STAGE1_ARG} \
+	${KUBELET_IMAGE} \
+		${KUBELET_IMAGE_ARGS} \
```

Comment:
This doesn't seem to be set anywhere. We can probably default to `--exec/kubelet`